data/budget.tsx

```diff
@@ -0,0 +1,43 @@
+export type BudgetRowType = {
+  expense: string;
+  description: string;
+  cost: number;
+};
+
+export const budgetRows: BudgetRowType[] = [
+  {
+    expense: 'Abonnement plateforme Meetup',
+    description: 'Plateforme pour publier nos événements, 90$ par semestre',
+    cost: 160,
```

Comment:
thinking: 90$ * 2 = 180, so is it related to the conversion to EUR ?

Reply:
Yes